pom.xml

```diff
@@ -123,6 +129,12 @@
       <version>1.4.7-jenkins-1</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>config-file-provider</artifactId>
+      <version>2.17.1-SNAPSHOT</version>
```

Comment:
You could use "maven deploy" to deploy a timestamped snapshot for testing this PR in CI